pytorch_blade/src/compiler/mlir/converters/mhlo_conversion.cpp

```diff
@@ -367,8 +368,7 @@ ConvertTorchToMhlo(std::shared_ptr<torch::jit::Graph> graph) {
         /*out*/ ::llvm::errs(),
         /*opPrintingFlags*/ print_flags);
   }
-  ::mlir::torch::TorchConversion::createTorchBackendToMhloBackendPipeline(
-      pm, options);
+  ::mlir::torch::createTorchBackendToMhloBackendPipeline(pm, options);
```

Comment:
Nit: how about renaming to `createDiscTorchBackendToMhloBackendPipeline` to distinguish it from the community `createTorchBackendToMhloBackendPipeline` that will be added.

Reply:
Good idea.